src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css

```diff
@@ -1,23 +1,32 @@
 .tokens {
   display: flex;
-  justify-content: center;
   align-items: center;
   margin-right: -3px;
   padding: 0 8px;
```

Comment:
This still doesn't render on the same line :cry: At least for me

![Screenshot from 2021-03-03 20-06-46](https://user-images.githubusercontent.com/1193222/109851142-01904680-7c5c-11eb-95b4-9498d1bd769c.png)

This is the change that makes it render on the same line for me:
```diff
diff --git a/src/modules/pages/components/RouteLayouts/UserNavigation.css b/src/modules/pages/components/RouteLayouts/UserNavigation.css
index 4e421823d..1f4fb51d6 100644
--- a/src/modules/pages/components/RouteLayouts/UserNavigation.css
+++ b/src/modules/pages/components/RouteLayouts/UserNavigation.css
@@ -98,7 +98,7 @@
 .walletAddress {
   display: flex;
   align-items: center;
-  padding: 0 8px;
+  padding: 2px 8px 0;
 
   /*
    * @NOTE We need to do border sizing shenanigans due to how the border color
diff --git a/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css b/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css
index 6bb2e06a0..bcfa77f52 100644
--- a/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css
+++ b/src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.css
@@ -2,7 +2,7 @@
   display: flex;
   align-items: center;
   margin-right: -3px;
-  padding: 0 8px;
+  padding: 0 8px 2px;
   height: 24px;
   position: relative;
   border: 1px solid transparent;
```

Please try it on your machine locally and if it works for you too, then add it in

Reply:
works fine as well, updated